services/web/server/tests/integration/02/test_computation.py

```diff
@@ -95,28 +95,10 @@ def __str__(self) -> str:
         return f"{self.__class__.__name__}({items})"
 
 
-def standard_role_response():
+def user_role_response():
     return (
         "user_role,expected",
         [
-            pytest.param(
```

Comment:
I really don't like this change.
Your reasoning for removing these tests is because there is a unit test that does the same thing. If that's the case then remove the slower test, but only if you are 100% that they test the exact same thing.

What I am seeing here is making the tests suite faster by removing tests. I do not feel comfortable with this approach.

Reply:
this part is only testing the user_role which is already tested in the unit test. the user_role is only checked when the entrypoint is called. After which no one cares about what the user_role is